dsc_lib/src/dscerror.rs

```diff
@@ -166,11 +167,25 @@ impl StreamMessage {
     pub fn print(&self, error_format:&StreamMessageType, warning_format:&StreamMessageType) -> Result<(), DscError>{
```

Comment:
This function is only used during discovery.  We should change that to use `error!` instead and remove the need for this entirely.

Reply:
Yes; that is the plan for discovery refactoring.

Reply:
Added TODOs to remove this during discovery refactoring.